internal/controller/vrg_volrep.go

```diff
@@ -105,8 +105,7 @@ func (v *VRGInstance) reconcileVolRepsAsPrimary() {
 		// subscription, such as, the deployment, pods, services, etc., by an
 		// entity external to the VRG a la IaC.
 		if err := v.uploadPVandPVCtoS3Stores(pvc, log); err != nil {
-			log.Info("Requeuing due to failure to upload PV object to S3 store(s)",
-				"errorValue", err)
+			log.Error(err, "Requeuing due to failure to upload PV object to S3 store(s)")
```

Comment:
Why log.Error? Dumping the call stack is not necessary. It will just overwhelm the logs.

Reply:
This is the only place we log errors in this code path. If we don't use log error, it will be hard to find the error. It tracebacks are the issue, we can configure the logger not to log tracebacks.

Reply:
But the error may be that we failed to annotate the PVC/PV - this is the error I have seen in CI. Hiding this error is not helpful.

I can send a logging change to disable the tracebacks in parallel. with this logging errors would not be an issue and it will easier to find issues.

Reply:
log.Error() does not log a stacktrace now.